src/AzureExtension/AzureExtension.cs

```diff
@@ -30,6 +34,8 @@ public AzureExtension(ManualResetEvent extensionDisposedEvent)
                 return new RepositoryProvider();
             case ProviderType.FeaturedApplications:
                 return new object();
+            case ProviderType.ComputeSystem:

```

Comment:
ComputeSystem?  I don't understand.  I thought this was for a VM.  Not a compute system.

Reply:
A compute system can be a virtual machine or a container, we decided to following the naming convention of the host compute system schema: https://learn.microsoft.com/en-us/virtualization/api/hcs/schemareference#computesystem. This allows us to keep generic naming since an environment can be a container. Thats the reasoning for the compute system terminology. In this case we're sending back a compute system provider. Compute systems for our purposes are generic and can be VM or a container